BuildRelease.msbuild

```diff
@@ -95,12 +95,30 @@
         <MSBuild Projects="@(ProjectToBuild)" Properties="Configuration=Release" />
     </Target>
     <Target Name="Test" DependsOnTargets="Build">
-        <xunitproject ProjectFile="Src\All.xunit" />
+        <PropertyGroup>
+          <Configuration Condition="'$(Configuration)' == ''">Release</Configuration>
+          <ParallelizeTests Condition="'$(ParallelizeTests)' == ''">true</ParallelizeTests>
+          <MaxParallelThreads Condition="'$(MaxParallelThreads)' == ''">0</MaxParallelThreads>
+        </PropertyGroup>
+
+        <ItemGroup>
+          <Xunit1TestAssemblies
+              Include="Src\*UnitTest\bin\$(Configuration)\*UnitTest.dll"

```

Comment:
How about using a pattern like `*Test.dll`? Will the proposed pattern (`*UnitTest.dll`) match

```
AutoFixtureDocumentationTest\bin\Release\Ploeh.AutoFixtureDocumentationTest.dll
```
?